ptest/usb_session.py

```diff
@@ -56,6 +59,16 @@ def __init__(self, device_name, uplink_console, port, is_teensy, simulation_run_
 
         # Open a connection to elasticsearch
         self.es = Elasticsearch([{'host':"127.0.0.1",'port':"9200"}])
+
+        #connect to email
+        self.username=tlm_config["email_username"]
+        self.password=tlm_config["email_password"]
+        self.mail = None
```

Comment:
This would prevent crashes if `tlm_config` entries weren't populated. My thought here is we wouldn't have to, for example, send our email password to the launch provider.
```suggestion
        #connect to email
        self.username=tlm_config.get("email_username", "")
        self.password=tlm_config.get("email_password", "")
        self.mail = None
```